cron/cron-staging.txt

```diff
@@ -0,0 +1,12 @@
+######################
+# CantusDB Cron Jobs #
+######################
+
+# Note: This is set up to run on the production server. If you want to run it on your local machine, you will need to change the paths.
+
+# min   hour    day     month   weekday command
+0       4       *       *       *       bash /home/ubuntu/code/CantusDB/cron/postgres/db_backup.sh
+10      4       *       *       *       bash /home/ubuntu/code/CantusDB/cron/management/manage.sh update_cached_concordances
+40      4       1       *       *       bash /home/ubuntu/code/CantusDB/cron/management/manage.sh populate_next_chant_fields; bash /home/ubuntu/code/CantusDB/cron/management/manage.sh populate_is_last_chant_in_feast
+# 50      4       *       *       7       /usr/local/bin/docker-compose -f /home/ubuntu/code/CantusDB/docker-compose.yml exec -T nginx lego --path /etc/nginx/ssl/live -d cantusdatabase.org -d www.cantusdatabase.org -d mass.cantusdatabase.org -m updateme@example.com --http --http.webroot /var/www/lego/ renew --days 45 --renew-hook "nginx -s reload" 
+50      4       *       *       7       /usr/local/bin/docker-compose -f /home/ubuntu/code/CantusDB/docker-compose.yml exec -T nginx lego --path /etc/nginx/ssl/live -d staging.cantusdatabase.org -d staging-alias.cantusdatabase.org -m updateme@example.com --http --http.webroot /var/www/lego/ renew --days 45 --renew-hook "nginx -s reload" 
```

Comment:
Could this run a single command that depends on a variable? (eg. If staging, then x; if production, then y)

Reply:
i.e., we create a new file, `update-certificates.sh` say, which takes an `--environment` argument say, with possible values `staging` and `production`, which would be called here?

Reply:
Yes, either with a keyword or required argument.

Reply:
Seriously, use Ansible. 

Deploy the staging cron file to staging. Deploy the production cron file to production. Template the paths/domains you need.